### PR TITLE
doc: add minutes for meeting 29 Oct 2019

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2019-10-29.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=GPScrh3EHDU
+* **GitHub Issue**: https://github.com/nodejs/build/issues/1990
+## Present
+
+* Rod Vagg
+* Michael Dawson (@mhdawson)
+* Sam Roberts
+* Ashely Cripps
```
Comment (BethGriggs) [inline doc/meetings/build-wg-meeting-2019-10-29.md]: *Ashley

File: doc/meetings/build-wg-meeting-2019-10-29.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+## Announcements
+ 
+*Extracted from **build-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+No items on agenda adhoc coverage of
+
+* Definition of role to capture what we need in terms of additional WG build members. Working
+  session to refine:  https://docs.google.com/document/d/1a2iLuj1Ga2FkdKHdGPAJUld1ibRkd-shl7lQJTD1KIw/edit
+  * Next steps:
+     * Michael will circulat to build WG members for additional comments
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2019-10-29.md]:
```suggestion
     * Michael will circulate to build WG members for additional comments
```

File: doc/meetings/build-wg-meeting-2019-10-29.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        mentioned that we should already have one. Michael to help Sam/Ashley get access to our
+        developer account to see if we can get what we need.
+  * Signing required for Catalina
+    * Currently it is possible to click through but unsure what will happen in Jan when Apple 
+      tightens things further
+    * Will affect older versions as well. Signing requires newer version of xcode though so it
+      would mean a toolchain upgrade for older releases which is not something we’d typically 
+      do.  Michael -> might be ok that if you run latest OSX version you can only run latest 
+      Node.js binaries. All -> good idea to open TSC issue as we currently don’t have a
+      Volunteer who has time to pick up this work and there are the larger questions (like is is
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2019-10-29.md]:
```suggestion
      Volunteer who has time to pick up this work and there are the larger questions (like is it
```

File: doc/meetings/build-wg-meeting-2019-10-29.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+
+  * Joyent migration
+    * We need to migrate machines at Joyent from one datacenter to another. Work is covered
+      so far as:
+        * Joao -> cross compilation machines
+        * Michael/Sam -> smartos machines
+        * Rod -> linting, infra machines and a couple other stragglers.
+
+     * At this point still work to but looks manageable.
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2019-10-29.md]:
```suggestion
     * At this point still work to do but looks manageable.
```

**Pull request merged**